src/cljs/org/broadinstitute/firecloud_ui/nih_link_warning.cljs

```diff
@@ -1,5 +1,6 @@
 (ns org.broadinstitute.firecloud-ui.nih-link-warning
   (:require
+   cljsjs.moment
```

Comment:
The NIH link warning banner checks isDbgapAuthorized ... it also needs to read from the nih status endpoint instead of the profile endpoint.